SPECS/ruby/ruby.spec

```diff
@@ -124,6 +122,8 @@ Provides:       rubygem-base64 = %{base64_version}-%{release}
 Provides:       rubygem(base64) = %{base64_version}-%{release}
 Provides:       rubygem-benchmark = %{benchmark_version}-%{release}
 Provides:       rubygem(benchmark) = %{benchmark_version}-%{release}
+Provides:       rubygen-bigdecimal = %{bigdecimal_version}-%{release}
```

Comment:
Please remove these provides for bigdecimal

Reply:
Oh, derp. Cleaned up the wrong package. One sec, will fix.

Reply:
Ok, should be fixed